### PR TITLE
Add a notebook that shows how best to use RNNs with skorch.

### PR DESCRIPTION
**Pull request by benjamin-work**

### CODE REVIEW

**Comment by benjamin-work** [comment]
@ottonemo ready for review

**Comment by ottonemo** [comment]
> First make sure to install an additional package, dstoolbox

This sounds like dstoolbox is mandatory for the use with skorch.

    out = F.softmax(self.output(drop))

Dimension parameter is missing.

**Comment by benjamin-work** [comment]
@ottonemo Points were addressed, pls review again.
